 
# Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  pan_controller:
    type: effort_controllers/JointPositionController
    joint: pan
    pid: {p: 50.0, i: 0.01, d: 1.0}
  tilt_controller:
    type: effort_controllers/JointPositionController
    joint: tilt
    pid: {p: 50.0, i: 0.01, d: 1.0}
